openfisca_us/variables/gov/states/mo/tax/income/subtractions/mo_qualified_health_insurance_premiums.py

```diff
@@ -0,0 +1,62 @@
+from openfisca_us.model_api import *
+
+
+class mo_qualified_health_insurance_premiums(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MO qualified healh insurance premiums"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://dor.mo.gov/forms/5695.pdf",  # Form MO 5695
+        "https://www.irs.gov/pub/irs-pdf/f1040sa.pdf",  # Federal Form 1040 Schedule A
+        "https://www.irs.gov/pub/irs-pdf/f1040.pdf",  # Federal Form 1040
+    )
+    defined_for = StateCode.MO
+
+    def formula(tax_unit, period, parameters):
+        # This logic is only required if we have a concept of premiums withheld from social security
+        # social_security_benefits = tax_unit("tax_unit_ssi", period)
+        # taxable_ss_benefits = tax_unit("tax_unit_taxable_social_security")
+        # taxable_ss_ratio = taxable_ss_benefits / social_security_benefits
+
+        person = tax_unit.members
+        # 'self_employed_health_insurance_premiums'
+
+        # Federal Schedule A, Line 1
+        med_dental_out_of_pocket = person(
+            "medical_out_of_pocket_expenses", period
+        )
+        total_health_insurance_premiums = add(
+            person, period, ["health_insurance_premiums"]
+        )  # total_health_insurance_premiums is also a primary input to the MO side of calculation, MO Form 5695, Line 8
+        # Federal Schedule A, Line 4
+        med_expense_deduction = tax_unit("medical_expense_deduction", period)
+
+        # the ratio of federal medical expense deduction to total medical expenses (out of pocket + premiums)
```

Comment:
Can these cancel out to `health_insurance_premiums + medical_out_of_pocket_expenses - medical_expense_deduction`? If so avoid the division, otherwise leave a comment explaining why.

Reply:
couldn't figure this out, sorry if i'm missing something. pretty sure the issue that we need med_dental_out_of_pocket only to calculate the ratio of federal_med_deduction/(med_dental_out_of_pocket+health_insurance_premiums), which is then used to scale the amount of premiums that can be claimed on this form. not sure how this could be reduced to addition and subtraction.